client/src/main/java/org/spine3/base/Stringifiers.java

```diff
@@ -90,4 +91,42 @@ private Stringifiers() {
                                     .convert(str);
         return result;
     }
+
+    /**
+     * Obtains {@code Stringifier} for map with default delimiter for the passed map elements.
+     *
+     * @param keyClass   the class of keys are maintained by this map
+     * @param valueClass the class  of mapped values
+     * @param <K>        the type of keys are maintained by this map
+     * @param <V>        the type of mapped values
+     * @return the stringifier for the map
+     */
+    public static <K, V> Stringifier<Map<K, V>> mapStringifier(Class<K> keyClass,
+                                                               Class<V> valueClass) {
+        checkNotNull(keyClass);
+        checkNotNull(valueClass);
+        final Stringifier<Map<K, V>> mapStringifier = new MapStringifier<>(keyClass, valueClass);
+        return mapStringifier;
+    }
+
+    /**
+     * Obtains {@code Stringifier} for map with custom delimiter for the passed map elements.
+     *
+     * @param keyClass   the class of keys are maintained by this map
+     * @param valueClass the class  of mapped values
+     * @param delimiter  the delimiter for the passed map elements via string
+     * @param <K>        the type of keys are maintained by this map
+     * @param <V>        the type of mapped values
+     * @return the stringifier for the map
+     */
+    public static <K, V> Stringifier<Map<K, V>> mapStringifier(Class<K> keyClass,
```

Comment:
Are the newly added `public static` methods covered with `null`-tolerance tests?